bin/control.sh

```diff
@@ -40,12 +40,19 @@ fi
 # Set SCRIPTS_HOME - base path to scripts.
 #
 SCRIPTS_HOME="${IGNITE_HOME_TMP}/bin"
+. "${SCRIPTS_HOME}"/include/functions.sh
```

Comment:
I am a bit confused about this change. IMHO, we need to add additional JVM options to the `CONTROL_JVM_OPTS` variable. Something like as follows:
```
CONTROL_JVM_OPTS=$(getJavaSpecificOptsForControlScript $version "$CONTROL_JVM_OPTS")
```
where `getJavaSpecificOptsForControlScript` should determine the required options based on the JVM version like the `getJavaSpecificOpts` does.

By the way, this fix should be applied to the `control.bat` script as well. Perhaps, it makes sense to update `snapshot-utility` scripts in the same way. WDYT?

Reply:
If I got your point right, you propose updating `getJavaSpecificOpts` instead of introducing a specific method for `control.sh`. Well, the idea looks good at first glance.